README.md

```diff
@@ -160,6 +160,9 @@ function getAllAddons({ categoryId }: GetAllAddonsParams = {}) {
   the keyboard. If it's something you think is unfixable then use
   `$FLOW_IGNORE` instead. Please explain your rationale in the comment and link
   to a GitHub issue if possible.
+* If you're stumped on why some Flow annotations aren't working, try using
```

Comment:
We don't rely on `flow type-at-pos` for anything. These docs are just giving a hint that the command exists and can be helpful. I don't think it makes sense to create `yarn` shortcuts for every single thing you can do with `flow`. There are a lot of things that are useful. For example, I use `flow stop` and `flow start` a lot to deal with the server. The number of `yarn` commands we have is already overwhelming which is itself problematic for contributors.